### PR TITLE
Add playbook for managing edxapp users

### PR DESCRIPTION
**Pull request by maxrothman**
@edx/devops

### CODE REVIEW

File: playbooks/edx-east/manage_edxapp_users_and_groups.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- hosts: all
+  vars:
+    python_path: /edx/bin/python.edxapp
+    manage_path: /edx/bin/manage.edxapp
+  tasks:
+    - name: Manage groups
+      shell: >
+        {{ python_path }} {{ manage_path }} lms manage_group {{ item.name }}
+        --permissions {{ item.permissions | default([]) | join(' ') }}
+        {% if item.get('remove') %}--remove{% endif %}
```
Comment (feanil) [inline playbooks/edx-east/manage_edxapp_users_and_groups.yml:74]: Why `item.get()` instead of `item.remove`?

Comment (maxrothman) [inline playbooks/edx-east/manage_edxapp_users_and_groups.yml:74]: Because `remove` is optional. If it's not defined, we don't want an error.

**Comment by feanil** [comment]
:+1:

**Comment by jibsheet** [comment]
:+1:

**Pull request merged**